### PR TITLE
Newsletter 13: Add a section about Egui

### PR DESCRIPTION
**Pull request by emilk**
https://github.com/rust-gamedev/rust-gamedev.github.io/issues/232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -66,6 +66,37 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+    }
+    ui.text_edit(&mut my_string);
+    ui.add(Slider::f32(&mut value, 0.0..=1.0).text("float"));
+});
+```
+
+![Egui](egui.png)
+
+_Discussions:
+[/r/rust](https://reddit.com/r/rust/comments/hzwvsk/emigui_deserves_more_love)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:94]:
```suggestion
[/r/rust](https://reddit.com/r/rust/comments/hzwvsk/emigui_deserves_more_love)_
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good, just missing a `_` to make the Discussions display in italics.

**Comment by ozkriff** [comment]
@emilk btw, could you enable that "allow edits from maintainers" checkbox in this PR? )

**Comment by ozkriff** [comment]
> I would if I could see it, but I don't:

Oh, that's weird :thinking:

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for updating, merged.

**Pull request merged**